addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -103,6 +103,23 @@ const StorePatch = {
     },
     onStarted() {
         super.onStarted(...arguments);
+        this.discuss.inbox = {
+            id: "inbox",
+            model: "mail.box",
+            name: _t("Inbox"),
+        };
+        this.discuss.starred = {
+            id: "starred",
+            model: "mail.box",
+            name: _t("Starred"),
+            counter: 0,
+        };
+        this.discuss.history = {
+            id: "history",
+            model: "mail.box",
+            name: _t("History"),
+            counter: 0,
+        };
```

Comment:
Can't it be done with default values? Not possible for relations? Or because of translate?

Reply:
Not possible for relations